src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java

```diff
@@ -1,70 +1,96 @@
 package edu.ucsb.cs156.gauchoride.config;
 
-import java.net.MalformedURLException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
+// import java.net.MalformedURLException;
+// import java.net.URI;
+// import java.net.URISyntaxException;
+// import java.net.URL;
+// import java.util.ArrayList;
+// import java.util.HashSet;
+// import java.util.List;
+// import java.util.Map;
+// import java.util.Optional;
+// import java.util.Set;
+// import java.util.regex.Pattern;
+
```

Comment:
Please remove the commented out code.

Reply:
Hi, 
My apologies, I deleted a different commented-out code block and missed that one. I commented it out to prevent accidentally deleting something I shouldn't. I have pushed my new changes. 

Is there a tool we could use to find commented out code blocks?